src/webhooks/README.md

```diff
@@ -0,0 +1,28 @@
+# Webhooks
+
+* Webhooks in "production" are deployed to a Google Cloud Run instance, in the project `super-big-data`. Why? (TODO insert why)
+* The webhook points to `https://product-eng-webhooks-vmrqv3f7nq-uw.a.run.app`
+
+## Adding a webhook to GoCD event emitter
+
+* goto [gocd](deploy.getsentry.net)
+* goto `admin` then `plugins`
+* find the `WebHook Notifier`
+* expand the text box by dragging the bottom right
+* add your URL
+
+## Creating a new webhook
+
+To create a new webhook, first create a new folder in this directory level with a name for the webhook. Next, create an `index.ts` file within this folder.
+
+Within this file, export a function with a unique name for the webhook that implements this function header: `export async function webhookName(request: FastifyRequest, reply: FastifyReply)`
```

Comment:
Should this say 'handler' rather than 'header'?

Reply:
No, it should be header since it has to have the fastifyreq and resp